Libraries/LibGfx/Painter.cpp

```diff
@@ -960,6 +1016,132 @@ void Painter::draw_text_line(const Rect& a_rect, const Utf32View& text, const Fo
     }
 }
 
+void Painter::draw_text_line(const Rect& a_rect, const Utf8View& text, const TTF::ScaledFont& font, TextAlignment alignment, Color color, TextElision elision)
```

Comment:
The number of duplications of this function is getting silly now. We need to find a way to share all this code between the slightly different implementations :)